src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -752,6 +739,26 @@ void ObjectMgr::LoadCreatureTemplate(Field* fields)
     creatureTemplate.ScriptID              = GetScriptId(fields[70].Get<std::string>());
 }
 
+/**
+* @brief (Re)loads the fast Creature Template cache from the main cache. Does not reload from the DB.
+*/
+void ObjectMgr::LoadCreatureTemplatesFastCache()
```

Comment:
I strongly considered making a second overload of this function that takes a single EntryId and only updates that. But since you'd have to iterate the entire _creatureTemplateStore to find it, I figured it's simpler and easier to just refresh the entire thing. The only times this should get called anyways is on server start and when the `.reload creature_template xxx` command is used, so performance impact should be virtually non-existant.